gorm/user.go

```diff
@@ -36,6 +36,18 @@ func (u *User) GetAllUser() ([]User, error) {
 	return users, err
 }
 
+// GetOneUser get one particular user based on id
```

Comment:
as mentioned above, this function is pretty much a duplicate of `PopulateByID()`. You can remove this

Reply:
Da xoa =))